python/src/pipeline/pipeline.py

```diff
@@ -1228,19 +1230,16 @@ def _write_json_blob(encoded_value):
   Returns:
     The blobstore.BlobKey for the file that was created.
   """
-  file_name = files.blobstore.create(mime_type='application/json')
-  handle = files.open(file_name, 'a')
-  try:
-    # Chunk the file into individual writes of less than 1MB, since the files
-    # API does not do buffered writes implicitly.
+  default_bucket = app_identity.get_default_gcs_bucket_name()
+  file_name = "/%s/appengine_pipeline/%s" % (default_bucket, str(uuid.uuid4()))
```

Comment:
Should we add the pipeline id to the prefix to further salt/organize the files written?